### DISCUSSION TITLE
Playlist frustrations

### DISCUSSION THREAD

**Discussion by kristophrk** [AzuraCast/AzuraCast#5625] [General]
I'm new to all of this but have a basic understanding of what I'm doing (I think/hope).  I have a station for all of my video game music.  I imported some 2400 odd songs.  I built a playlist and reshuffled it.  But every time it plays a song from a particular album, it plays the entire album (in this case its Fallout 4).  Which is a great album but I don't want to hear the entire album like this.  So again, I built a new play list and even reshuffled it twice - but continues to play random songs - but when it hits the Fallout 4 album - it plays them all together.  And yes - they are separate mp3 files so I cannot figure out why this is happening?  Anyone else have this issue?

**Comment by kristophrk** [comment]
It keeps happening so I just deleted the files.  i'd still like to know why this is tho.  Literally every other album gets shuffled randomly - but fallout 4 keeps getting clumped together.

**Reply by paulcjkp** [reply]
Do you have Avoid Duplicate Artists turned on?  Is the playlist set to Shuffle, or Random?
